src/holofuel/components/layout/PrimaryLayout/PrimaryLayout.js

```diff
@@ -24,13 +27,36 @@ export function PrimaryLayout ({
   const { loading: holofuelUserLoading, data: { holofuelUser = {} } = {} } = useQuery(HolofuelUserQuery)
   const { loading: ledgerLoading, data: { holofuelLedger: { balance: holofuelBalance } = {} } = {} } = useQuery(HolofuelLedgerQuery, { fetchPolicy: 'cache-and-network' })
 
+  const [hasCalledReroute, setHasCalledReroute] = useState(false)
+
   const inboxCount = actionableTransactions.filter(actionableTx => actionableTx.status !== STATUS.canceled && !((actionableTx.status === STATUS.declined) && (actionableTx.type === TYPE.request))).length
 
   const isWide = useContext(ScreenWidthContext)
   const [isMenuOpen, setMenuOpen] = useState(false)
   const hamburgerClick = () => setMenuOpen(!isMenuOpen)
   const handleMenuClose = () => setMenuOpen(false)
 
+  const { push } = useHistory()
+  // eslint-disable-next-line
```

Comment:
this is disabling a `missing dependencies' warning?

Reply:
Exactly. `react-hooks/exhaustive-deps` :100: 

Not a standard, nor a preference, just apart of the unique setup and temporary solutions, as outlined below.